internal/menu.go

```diff
@@ -0,0 +1,11 @@
+package internal
+
+import (
+	"time"
+)
+
+type Menu struct {
+	Nombre string
+	Fecha time.Time
+	Platos []Plato
```

Comment:
¿Qué ocurre si añadimos 6 platos a este menú?

Reply:
@antoniorr02 todavía no has aprobado explícitamente esto, y *debéis hacerlo*. Vais a tener que trabajar con este código. ¿Podríais, por favor, ver si se ajusta a los milestones e HUs *que vosotros mismos habéis puesto*?
Y cuando digo aprobar explícitamente, digo que lo revises bien, que se hagan los cambios convenientes, y *entonces* volvéis a pedirme revisión.

Reply:
Considero que con poder introducir platos cuando en un futuro se extraigan de los diferentes menús la estructura de cada menú dentro de cada comedor definirá cuantos platos son necesarios en cada menú, con lo que pienso que dar flexibilidad en este aspecto es esencial.

Si bien, poner un límite de 5 platos como ha propuesto @MarioRgzLpz  creo que buena idea, ya que no tendría sentido un menú con una mayor cantidad de platos.

En tal caso, en ese aspecto tiene mi aprobación.

Por otro lado, lo que si echo en falta es lo que se ha eliminado para poder guardar los comedores favoritos, ya que se comenta en la HU1, #4.
Por lo demás, hasta donde llega mi conocimiento sobre el problema no tengo ninguna pega adicional. Si bien, estoy abierto a escuchar sugerencias de aspectos que probablemente se me escapen.

Reply:
Entiendo lo que @antoniorr02 quiere modelar pero, en el Milestone 1 no habla nada de comedores favoritos y como en la asignatura solo vamos a desarrollar hasta dicho Milestone no veo necesario para que sea mínimo realizar esta implementación. De igual manera si esto no es así y si que hace falta no tendría problema en añadirlo de nuevo.

Reply:
Hay que resolver *todos* los comentarios.